charts/galley/templates/configmap.yaml

```
@@ -25,6 +25,11 @@ data:
       tlsCa: /etc/wire/galley/cassandra/{{- (include "tlsSecretRef" . | fromYaml).key }}
       {{- end }}
 
+    postgresql: {{ toYaml .postgresql | nindent 6 }}
+    {{- if hasKey $.Values.secrets  "pgPassword" }}
```

```suggestion
    {{- if hasKey $.Values.secrets "pgPassword" }}
```

Nitpicking 🤓 
Superfluous whitespace.